include/deal.II/numerics/error_estimator.templates.h

```diff
@@ -476,7 +476,7 @@ namespace internal
         for (unsigned int component=0; component<n_components; ++component)
           if (parallel_data.component_mask[component] == true)
             for (unsigned int p=0; p<n_q_points; ++p)
-              face_integral[n] += Utilities::fixed_power<2>(parallel_data.phi[n][p][component]) *
+              face_integral[n] += std::norm(parallel_data.phi[n][p][component]) *
```

Comment:
Let's not quite do that yet. Can you just call `std::abs(...)` on the argument of `fixed_power`?

Alternatively (and maybe more elegantly), we have `NumberTraits::abs_square()` which I think is what you want.